operations/helm/charts/grafana-agent/templates/_helpers.tpl

```diff
@@ -55,9 +55,7 @@ helm.sh/chart: {{ include "grafana-agent.chart" . }}
 app.kubernetes.io/version: "vX.Y.Z"
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- else }}
-{{- if .Chart.AppVersion }}
-app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
-{{- end }}
+app.kubernetes.io/version: {{ substr 1 -1 (include "grafana-agent.imageId" .) }}
```

Comment:
What's the substr call here for, getting rid of quotes? Do we need it?

Reply:
it's to get rid of the delimiter character that constitutes the first character of the output of the `"grafana-agent.imageId"` template function. For example, for image tags, instead of `v0.37.4`, `"grafana-agent.imageId"` alone would output `:v0.37.4`. For digests, the output would be `@sha256:<digest>` instead of `sha256:<digest>`.

Reply:
Makes sense!